getting-started/repl/intergration.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Integration
+page_title: Telerik REPL for Blazor Integration
+description: Telerik REPL for Blazor integration in other Telerik Resources, Visual Studio and Visual Studio Code
+slug: blazor-repl-integration
+tags: telerik,blazor,repl,integration
+published: True
+position: 10
+---
+
+# Telerik REPL for Blazor Integration
+
+Telerik REPL for Blazor is integrated in our resources (documentation and demos), as well as Visual Studio and Visual Studio Code.
+
+In this article:
+
+* [Telerik UI for Blazor documentation Integration](#telerik-ui-for-blazor-documentation-integration)
+* [Telerik UI for Blazor demos Integration](#telerik-ui-for-blazor-demos-integration)
+* [Visual Studio and Visual Studio Code Integration](#visual-studio-and-visual-studio-code-integration)
+
+## Telerik UI for Blazor Documentation Integration
+
+The Telerik UI for Blazor documentation supports integration with the Telerik REPL for Blazor. This provides interactive experience and easy to test snippets for the various components and scenarios. Above the code snippets listed in the documentation articles you will find a toolbar with the following options:
+
+* `Edit` - click to open the source in the integrated the Telerik REPL for Blazor Editor.
+
+* `Preview` - click to run the snippet in the integrated the Telerik REPL for Blazor Result View.
+
+
+![Documentation Integration](../images/repl-docs-integration.png)
+
+
+## Telerik UI for Blazor Demos Integration
+
+The Telerik UI for Blazor demos support integration with the Telerik REPL for Blazor. In the demo window toolbar, along with the "Example" and "View Source" tabs, you will find a tab with dropdown that provides option for editing the demo in Telerik REPL for Blazor. Once you select that option, you will be redirected to the Telerik REPL for Blazor and its editor will contain the source code of the current demo, so you can directly edit and test it.
```

Comment:
We are talking about another website. Why not link to it, so that the reader can see for themselves.

Reword the "you will find" part.